src/revpi-functions

```diff
@@ -9,3 +9,38 @@ cm1-detection() {
     grep -m 1 "model name" /proc/cpuinfo | grep -q v6l
     return $?
 }
+
+# Check existance of HAT EEPROM
+hats-detection() {
+	if [ -d /proc/device-tree/hat ]; then
+		return 1;
+	fi
+	return 1;
+}
+
+# Revert configurations in config.txt made by revpi-factory-reset
+revert-config-if-hats () {
```

Comment:
Why is this in the revpi-functions? This will not be used in the revpi-factory-reset.sh

Reply:
And it is not used here either!

Reply:
just think as we have a separate script for function,  all function should be put here. 

It will be used in "revpi-factory-reset"

Reply:
Why scatter the functions over the files if it is not necessary?

Reply:
get it, would just know the choise.
BTW, I think the function revert-config-if-hats is not needed, as revert is always done before the "reset".